docs/tutorial/units.rst

```diff
@@ -4,82 +4,133 @@
-  >>> length
+  >>> displacement = distance_end - distance_start
+  >>> displacement
   <Quantity 22.99999 km>
 
   >>> time = 15 * u.minute
-  >>> speed = length / time
+  >>> speed = displacement / time
   >>> speed
   <Quantity 1.53333267 km / min>
 
-However, operations which do not make physical sense for the units specified will cause an error::
+However, operations with incompatible units raise an error::
 
-  >>> length + time
+  >>> displacement + time
   Traceback (most recent call last):
   ...
   astropy.units.core.UnitConversionError: Can only apply 'add' function to quantities with compatible dimensions
 
+Converting Units
+================
+
+These `~astropy.units.Quantity` objects can also be converted to other units or unit systems::
+
+  >>> length.to(u.km)
+  <Quantity 0.01 km>
+
+  >>> length.cgs
+  <Quantity 1000. cm>
+
+Unit Equivalencies
+==================
+
+It is commonplace to convert between units which are only compatible under certain assumptions.
+For example, in spectroscopy, spectral energy and wavelength are equivalent given the relation
+:math:`E=hc/\lambda`.
+If we try to convert a wavelength to energy using what we learned in the previous section,
+we get an exception because length and energy are, in general, not compatible units,::
+
+  >>> length.to(u.keV)
+  Traceback (most recent call last):
+  ...
+  astropy.units.core.UnitConversionError: 'm' (length) and 'keV' (energy/torque/work) are not convertible
+
+However, we can perform this conversion using the `~astropy.units.equivalencies.spectral` equivalency,::
+
+  >>> length.to(u.keV, equivalencies=u.spectral())
+  <Quantity 1.23984198e-10 keV>
+
+An equivalency common in solar physics is conversion of angular distances in the plane of the sky to physical distances on the Sun.
+To perform this conversion, sunpy provides `~sunpy.coordinates.utils.solar_angle_equivalency`, which requires specifying the location at which that angular distance was measured::
+
+  >>> from sunpy.coordinates import get_earth
+  >>> from sunpy.coordinates.utils import solar_angle_equivalency
+  >>> length.to(u.arcsec, equivalencies=solar_angle_equivalency(get_earth("2013-10-28")))
+  INFO: Apparent body location accounts for 495.82 seconds of light travel time [sunpy.coordinates.ephemeris]
+  <Quantity 1.38763748e-05 arcsec>
+
+Note that in the above example we made use of `sunpy.coordinates.get_earth`.
+We will talk more about coordinates in the :ref:`sunpy-coordinates` section of this tutorial.
+For now, it is just important to know that this function returns the location of the Earth on 2013 October 28.
+
+
+Dropping Units
+==============
+
+Not every package in the scientific Python ecosystem understand units.
+As such, it is sometimes necessary to drop the units before passing `~astropy.units.Quantity` to such functions.
+As shown above, you can retrieve the just the numerical value of a `~astropy.units.Quantity`,::
+
+  >>> length.to_value()
```

Comment:
```suggestion
  >>> length.value
```
To be consistent with the top of the tutorial?

Reply:
I changed this to `to_value` at the suggestion of @Cadair. I think it is fine to show both? Maybe it would be better to show that it can be used to convert to different unit systems and get just the numerical value in one function call.